src/DynamoPackages/PackageManagerSearchElement.cs

```diff
@@ -155,6 +156,7 @@ public void Upvote()
                 , TaskScheduler.FromCurrentSynchronizationContext());
         }
 
+        [Obsolete("This API will no longer decrease package votes and will be removed in Dynamo 3.0")]
         public void Downvote()
```

Comment:
if someone calls this method - it should do nothing right? Will the continueWith ever get called?

Reply:
Yes, since the handler of the `DownvoteRequested` event is removed, the call will do nothing and return nothing